man/journald.conf.xml

```diff
@@ -139,6 +139,50 @@
         <literal>us</literal>. To turn off any kind of rate limiting,
         set either value to 0.</para>
 
+        <para>The <varname>RateLimitBurst=</varname> is further modified
+        by the log() of the available disk space for the journal.</para>
+
```

Comment:
i think it's fine to document the multiplier, but I think we should somehow indicate that this is the current implementation and that this should not be considered API. Maybe just add a "currently" to the sentence. Maybe:

"Note that the effective rate limit is multiplied with a factor derived from the available disk space for the journal. Currently, this factor is calculated from the logarithm to the base of 2, …:"

Reply:
OK. But the current behavior is completely unexpected based on the existing documentation.